dashboard/app/helpers/lesson_import_helper.rb

```diff
@@ -0,0 +1,213 @@
+# Helper module for importing data from curriculumbuilder
+module LessonImportHelper
+  # Lockable lessons don't need to be merged with curriculumbuilder, but we do
```

Comment:
There are CSP lessons where a lockable lesson has a lesson plan. https://www.codecurricula.com/csp-20/unit4/15/

Reply:
This should now be addressed between Dave's recent changes and 75f2de5e0897db75e0c4597f5c889d55bef85289.